website/content/en/highlights/2022-10-04-0-25-0-upgrade-guide.md

```diff
@@ -126,8 +128,14 @@ will be removed in the future.
 | delete | remove_metadata_field(.foo.bar)       | del(%foo.bar)      |
 
 
-#### Deprecation of `endpoint` option in Elasticsearch sink
+#### Deprecation of `endpoint` option in Elasticsearch sink {#elasticsearch-endpoint-deprecation}
 
 Vector `0.25.0` has introduced distribution of events to multiple endpoints for Elasticsearch sink.
 In order to enable this distribution, a new `endpoints` setting has been introduced which configures one or more destinations to which to distribute the events.
 The existing `endpoint` setting is now deprecated and will be removed in a future version.
+
+#### Deprecation of the Lua version 1 API {#lua-v1-api-deprecation}
+
+Vector `0.9.0` introduced the version 2 API for the `lua` transform.
+This API has long been considered fully mature, obviating the need to maintain the older API.
+The older version 1 API is now deprecated and will be removed in a future version.
```

Comment:
Can we add a migration example? AKA a config using version 1 and how to convert that to version 2?

Reply:
Good idea. I'll dig up an old script to convert it.